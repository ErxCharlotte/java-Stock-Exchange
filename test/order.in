ADD trader1 100.00
ADD trader2 100.00
ORDER 0000
IMPORT trader1 ABC 100.00
IMPORT trader1 CDE 500.00
SELL trader1 ABC 100 5
ORDER 1000
ORDER 0000
BUY trader2 LOL 100 1
BUY trader2 LOL 100 2
BUY trader2 LOL 100 3
BUY trader2 LOL 100 4
BUY trader2 LOL 100 5
BUY trader2 LOL 100 6
BUY trader2 LOL 100 7
BUY trader2 LOL 100 8
BUY trader2 LOL 100 9
BUY trader2 LOL 100 10
BUY trader2 LOL 100 11	
ORDER 0005
ORDER 000A
SELL trader1 CDE 100 12
ORDER 000C
EXIT