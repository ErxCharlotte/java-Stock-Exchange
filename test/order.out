$ Success.
$ Success.
$ No orders in either book in the market.
$ Trader now has 100.00 units of ABC.
$ Trader now has 500.00 units of CDE.
$ No trades could be made, order added to sell book.
$ Order is not present in either order book.
$ 0000: SELL 100.00xABC @ $5.00
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ 0005: BUY 100.00xLOL @ $5.00
$ 000A: BUY 100.00xLOL @ $10.00
$ No trades could be made, order added to sell book.
$ 000C: SELL 100.00xCDE @ $12.00
$ Have a nice day.